### PR TITLE
Improved GitHub issues layout

### PR DESCRIPTION
**Pull request by gastaldi**
This improves issues' readability. Example: https://github.com/quarkusio/quarkus/issues/15679

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -7,42 +7,42 @@ assignees: ''
...
 
 Or attach an archive containing the reproducer to the issue.
 
 Steps to reproduce the behavior:
-1. 
-2. 
-3. 
+1.
+2.
+3.
```
Comment (gsmet) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: I would have kept the spaces here.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -7,42 +7,42 @@ assignees: ''
...
- - Output of `java -version`: 
- - GraalVM version (if different from Java): 
- - Quarkus version or git rev: 
- - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 
+## Environment (please complete the following information):
+- Output of `uname -a` or `ver`:
+- Output of `java -version`:
+- GraalVM version (if different from Java):
+- Quarkus version or git rev:
+- Build tool (ie. output of `mvnw --version` or `gradlew --version`):
```
Comment (gsmet) [inline .github/ISSUE_TEMPLATE/bug_report.md:45]: Same here.

**Review by gsmet — CHANGES_REQUESTED** [review]
Good idea but I wouldn't remove the trailing spaces in this case.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (gastaldi) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: I blame IDEA for that 😝

**Comment by gastaldi** [comment]
I made the Environment section in the Bug Report template a bit different, using sections instead of single line

**Review by gsmet — APPROVED** [review]

**Pull request merged**